src/tokens.js

```diff
@@ -360,6 +372,13 @@ const tokens = {
       }
     }
 
+    if (isNFT) {
+      // After the transaction data is completed
+      // if it's an NFT I must add the first output as the data script
+      // For NFT data the value is always 1 HTR
+      txData.outputs.unshift({ data: nftData, value: 1, tokenData: 0 });
```

Comment:
This can be any string. The full node has the standard output size validations but I don't think we should validate anything here. Maybe the user is creating a non standard output and will mine/propagate in his own node.